app/src/main/java/com/orange/ouds/app/ui/tokens/TokenCategoryDetailScreen.kt

```diff
@@ -46,35 +47,53 @@ fun TokenCategoryDetailScreen(tokenCategory: TokenCategory) {
                 Spacer(modifier = Modifier.height(OudsSpacingFixedKeyToken.Medium.value))
             }
 
-            items(tokens) { token ->
-                Row(
-                    modifier = Modifier
-                        .fillMaxWidth()
-                        .padding(horizontal = OudsSpacingFixedKeyToken.Medium.value, vertical = OudsSpacingFixedKeyToken.Shorter.value)
-                ) {
-                    when (tokenCategory) {
-                        is TokenCategory.Opacity -> tokenCategory.Illustration(opacity = token.value as Float)
-                        is TokenCategory.Elevation -> tokenCategory.Illustration(elevation = token.value as Dp)
-                    }
-                    Column(
+            items(tokenCategory.properties) { tokenProperty ->
+                tokenProperty.nameRes?.let {
+                    Text(
                         modifier = Modifier
-                            .weight(1f)
-                            .padding(start = OudsSpacingFixedKeyToken.Medium.value)
+                            .fillMaxWidth()
+                            .padding(OudsSpacingFixedKeyToken.Medium.value),
```

Comment:
In Figma there is a space which is two times `OudsSpacingFixedKeyToken.Medium` between the last row of a token property and the next token property title.